packages/backend/src/dbt/profiles.ts

```diff
@@ -174,6 +174,24 @@ const credentialsTarget = (
                     [envVar('token')]: credentials.personalAccessToken,
                 },
             };
+        case WarehouseTypes.ATHENA:
+            return {
+                target: {
+                    type: WarehouseTypes.ATHENA,
+                    // all dbt config
+                    work_group: credentials.workgroup,
+                    region_name: credentials.awsRegion,
+                    s3_staging_dir: credentials.outputLocation,
+                    schema: credentials.schema,
+                    database: credentials.database,
+                    aws_access_key_id: credentials.awsAccessKeyId,
+                    aws_secret_access_key: credentials.awsSecretKey,
+                },
+                environment: {
+                    [envVar('AWS_ACCESS_KEY_ID')]: credentials.awsAccessKeyId,
+                    [envVar('AWS_SECRET_KEY')]: credentials.awsSecretKey,
```

Comment:
the right ENV variable should be AWS_SECRET_ACCESS_KEY, see [this](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)